generator/views.py

```diff
@@ -71,7 +71,7 @@ def form_valid(self, form):
                    'spoiler_log': RandomizerInterface.get_web_spoiler_log(pickle.loads(game.configuration)),
                    'is_race_seed': game.race_seed,
                    'share_info': share_info.getvalue()}
-        return render(self.request, 'generator/seed.html', context)
+        return redirect('/share/' + game.share_id)
```

Comment:
We can remove lines 66-73 here since we aren't using the share and context information in the redirect.

Reply:
ah, good point. I'm too used to languages/systems which yell at me when I don't use variables clearly. I'll fix it